epuck_ros2_cpp/src/driver.cpp

```diff
@@ -236,11 +244,15 @@ class EPuckPublisher : public rclcpp::Node {
   void publishDistanceData(rclcpp::Time &stamp) {
     // Decode measurements
     float dist[8];
+    float distTof = OUT_OF_RANGE;
     for (int i = 0; i < 8; i++) {
       const int distanceIntensity = mMsgSensors[i * 2] + (mMsgSensors[i * 2 + 1] << 8);
-      float distance = EPuckPublisher::intensity2distance(distanceIntensity) + SENSOR_DIST_FROM_CENTER;
+      float distance = EPuckPublisher::intensity2distance(distanceIntensity);
       dist[i] = distance;
     }
+    if (mTofInitialized) {
+      distTof = tofReadDistance() / 1000.0;
+    }
```

Comment:
```suggestion
    if (mTofInitialized)
      distTof = tofReadDistance() / 1000.0;
```
https://github.com/cyberbotics/webots/wiki/CPP-Coding-Style#cs360-single-if-while-or-for-statement-must-be-written-without-brackets-the-statement-part-should-be-put-on-a-separate-line-in-case-of-nested-block-keep-brackets